src/client/pages/OfferNew/types.ts

```diff
@@ -73,7 +73,9 @@ export type GenericQuoteData = {
   subType?: string
   apartment?: string
   floor?: number
+  squareMeters?: number
 
+  numberOfWetUnits?: number
   ancillaryArea?: number
   numberOfBathrooms?: number
   yearOfConstruction?: number
```

Comment:
House NO quote creation also involves providing `yearOfOwnership`. I'm not sure if we should also display that tho

Reply:
good point. I will ask @zakkjellin about this.